.env

```diff
@@ -0,0 +1,5 @@
+SERVER_PUBLIC_DOMAIN="http://didcomm-mediator.com"
+SERVER_LOCAL_PORT="8080"
+STORAGE_DIRPATH="crates/generic-server/target/storage"
+MONGO_DBN="mediator-coordination"
+MONGO_URI="mongodb://localhost:27017/"
```

Comment:
It's important to keep the database URL private because, in the final production, it will contain a username and password.